qiita_pet/test/test_portal.py

```diff
@@ -0,0 +1,58 @@
+from unittest import main
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+from mock import Mock
+
+from qiita_db.user import User
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class TestPortal(TestHandlerBase):
+    def test_get(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.get('/admin/portals/studies/')
+        self.assertEqual(response.code, 200)
+
+    def test_post(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.post('/admin/portals/studies/', {'portal': 'EMP',
+                                                         'selected': [1],
+                                                         'action': 'Add'})
+        self.assertEqual(response.code, 200)
+
+        response = self.post('/admin/portals/studies/', {'portal': 'EMP',
```

Comment:
Out of curiosity, why do you think is needed? Note that these are really simple tests and seem like too much to have a test for each case, for example take a look at test_*_errors